### PR TITLE
chore(test): add tpcds to test

### LINKED ISSUE #10579: Add tpcds to tests
**Summary**

- [x] Add TPC-DS to tests (data scale: sf=0.01). #10673
- [x] Pass all queries in standalone mode.
- [ ] Pass all queries in cluster mode.

### PR DESCRIPTION
**Pull request by xudong963**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

to avoid draw back

Tacking in #10579

### CODE REVIEW

File: tests/suites/0_stateless/18_tpcds/18_0002_q2.sql
Comment (xudong963) [inline tests/suites/0_stateless/18_tpcds/18_0002_q2.sql:1]: Q2: ERROR 1105 (HY000) at line 1: Code: 1104, displayText = attempt to divide by zero.

It's dataset-sensitive @sundy-li @RinChanNOWWW

**Comment by xudong963** [comment]
@RinChanNOWWW  will clean dataset to avoid divide 0, after dataset is updated, let continue the ticket.

**Comment by RinChanNOWWW** [comment]
> @RinChanNOWWW will clean dataset to avoid divide 0, after dataset is updated, let continue the ticket.

Used an alternative way: modify the SQLs (`where` conditions) to hit valid data.

File: tests/suites/0_stateless/18_tpcds/18_0061_q61.sql [hunk 1]
```diff
@@ -32,11 +31,9 @@ from
    and   ss_customer_sk= c_customer_sk
    and   ca_address_sk = c_current_addr_sk
    and   ss_item_sk = i_item_sk
-   and   ca_gmt_offset = -6
+   and   ca_gmt_offset = -0.06
```
Comment (sundy-li) [inline tests/suites/0_stateless/18_tpcds/18_0061_q61.sql:34]: sqlogictest support template by {sf}.

Maybe we can add this in `databend-test.`

**Review by RinChanNOWWW — APPROVED** [review]

**Pull request merged**